src/builtin.d

```diff
@@ -150,10 +151,10 @@ extern (C++) Expression eval_yl2x(Loc loc, FuncDeclaration fd, Expressions* argu
     assert(arg0.op == TOKfloat64);
     Expression arg1 = (*arguments)[1];
     assert(arg1.op == TOKfloat64);
-    real x = arg0.toReal();
-    real y = arg1.toReal();
-    real result;
-    Port.yl2x_impl(&x, &y, &result);
+    real_t x = arg0.toReal();
```

Comment:
Global comment: use `auto` or (even better) `const` where possible instead of `real_t`.